### PR TITLE
Update documentation to create SaaS policies

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR updates the documentation for creating and installing new SaaS policies.

### CODE REVIEW

File: docs/intern/policies/index.rst [hunk 1]
```diff
@@ -1,12 +1,66 @@
...
+
+- Aktuellen Release im `versions.cfg` eintragen
+- Initiale Templates hinzufügen, falls vorhanden. Diese müssen in der Datei `opengever.[name]/opengever/[name]/profiles/default_content/opengever_content/02-templates.json` eingetragen werden, die entsprechenden blobs kommen in `/opengever.[name]/opengever/[name]/profiles/default_content/opengever_content/templates`
+- Ordnungssystem gemäss :ref:`policies-repository` hinzufügen
+- `bin/buildout` mit `development.cfg`
+- Plone-Site erstellen
+- Theme übers `@@teamraumtheme-controlpanel` anpassen, Logo und u.U. `Höhe des Kopfbereiches`
+- Theme exportieren und als `opengever.[name]/opengever/[name]/profiles/default/customstyles.json` hinterlegen[name]/opengever/[name]/profiles/default/customstyles.json` hinterlegen
+- Finaler lokaler Test, Plone-Site löschen und neu erstellen
+- Repo `http://github.com/4teamwork/opengever.[name]` einrichten und Policy pushen
```
Comment (deiferni) [inline docs/intern/policies/index.rst:47]: privates

File: docs/intern/policies/index.rst [hunk 2]
```diff
@@ -1,12 +1,66 @@
...
+- Domain: [name].onegovgever.ch *z.B.: 4teamwork.onegovgever.ch*
+- Repo-Link: https://github.com/4teamwork/opengever.[name]
+- Server: [name] *z.B.: ipet*
+- Server-Pfad:
+- Deployment: [number] *z.B.: 09*
+- Plone-Site id: [id] *z.B.: 4tw*
+- postgres-db: ogds_[name] *z.B.: ogds_4teamwork*
+- LDAP Ast `ou=[name],ou=OneGovGEVER,dc=4teamwork,dc=ch` einrichten
+- Bumblebee-Name: gever_[name]
+
```
Comment (deiferni) [inline docs/intern/policies/index.rst:24]: bumblebee secret

File: docs/intern/policies/index.rst [hunk 3]
```diff
@@ -1,12 +1,66 @@
...
+- `bin/buildout` mit `development.cfg`
+- Plone-Site erstellen
+- Theme übers `@@teamraumtheme-controlpanel` anpassen, Logo und u.U. `Höhe des Kopfbereiches`
+- Theme exportieren und als `opengever.[name]/opengever/[name]/profiles/default/customstyles.json` hinterlegen[name]/opengever/[name]/profiles/default/customstyles.json` hinterlegen
+- Finaler lokaler Test, Plone-Site löschen und neu erstellen
+- Repo `http://github.com/4teamwork/opengever.[name]` einrichten und Policy pushen
+- Initiale Gruppen: e.g.: [name]_admins, [name]_users, [name]_inbox im LDAP erstellen (https://ldaptain.4teamwork.ch/)
+
+Policy auf Server installieren
+------------------------------
```
Comment (deiferni) [inline docs/intern/policies/index.rst:51]: i think you dropped the following points, maybe amend/rewrite and amend?

```
Auf dem zu deployenden repository ist die policy zu clonen.
Achte darauf, dass der Name des Zielordners dem Muster der anderen Deployments entspricht.
Vergiss nicht, die Postgresql Datenbank mit ``createdb`` zu erstellen.

Führe danach die Installation des GEVERs aus.
Der Pfad zum Python-Executable kann einer Instanz einer bereits eingerichteten policy entnommen werden.

Nach dem Buildout kann ``bin/supervisord`` gestartet werden und mit ``bin/supervisorctl`` die einzelnen Instanzen gestartet werden.

Das GEVER kann nun mittels SSH-Tunnel noch installiert werden.
```

Comment (elioschmutz) [reply docs/intern/policies/index.rst:51]: @deiferni I'ts too much text for something which should be known by everyone (supervisor, ssh and cloning the policy). But I extended the the documentation with the ``createdb`` command which seems to be an important notice.

Comment (deiferni) [reply docs/intern/policies/index.rst:51]: yeah i did not mean to literally copy it, but i'd suggest to mention the following points:
- createdb
- follow folder naming convention on the server
- make sure to use correct python path for bootstrapping
- make sure to boot instances with supervisord
- gever can then be installed via ssh-tunnel to instance port

sure, those things are known to us but the docs could/should also be for new developers 😉.

Comment (elioschmutz) [reply docs/intern/policies/index.rst:51]: 👍 I extendet the list

**Pull request merged**